projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js

```diff
@@ -32,9 +32,11 @@ export const settings = {
 		const [ dontShowAgain, setDontShowAgain ] = useState( false );
 		const [ isChecked, setIsChecked ] = useState( false );
 
-		const { launchpadScreenOption, siteIntentOption } = window?.Jetpack_LaunchpadSaveModal || {};
+		const { launchpadScreenOption, hasNeverPublishedPostOption, siteIntentOption } =
+			window?.Jetpack_LaunchpadSaveModal || {};
```

Comment:
I did not know that there was a Jetpack_LaunchpadSaveModal property on the window object. This seems like a strange way to store and retrieve application state, but I assume it's just a Jetpack thing.

Reply:
I'm not entirely sure either. Maybe @Sharpirate or @jeherve can elaborate more on this?

Reply:
My understanding is that this is a common way in Jetpack to hydrate the FE with the site options we're reading in the php file

Reply:
Thanks! If this is an established convention, it raises a question I asked elsewhere in the comments - do they already hydrate site_intent the same way?

Reply:
https://github.com/Automattic/jetpack/pull/28989#discussion_r1108842297